testing/regress/ecl/split_test.ecl

```diff
@@ -511,8 +511,6 @@ res14 := gatherCounts(dst14[1].targetFilename, origRecCount, dst14[1].noSplit);
 
 sequential (
 #if (VERBOSE = 1)
-    output(pad),
-    output(length(pad)),
```

Comment:
is change related/intentional ?

Reply:
This test failed in Smoketest during rework and executed in local environment. When verbose was enabled this test didn't compile based on that missing reference (pad).